README.md

```diff
@@ -34,46 +34,53 @@ sudo usermod -a -G docker $USER
 
 You will then need to exit the ssh session and log in again.
 
-Finally, you will need to create the `.env` configuration file:
+Install ETH Docker.
 
 ```
-cd ssv-validator-node
-cp .env.sample .env
+cd eth-docker
+./ethd install
 ```
 
-Edit the `.env` in your favourite editor and set the variable `NETWORK=holesky`.
-
-## Generating Operator Keys
-
-All cluster members will need an encrypted key pair to run their node. To create the operator key pair the operator first generates an encryption password:
+Enable ethd to be called from anywhere on your terminal.
 
 ```
-mkdir secrets
-tr -dc 'A-Za-z0-9' </dev/urandom | head -c 16 >> secrets/password
+source ~/.profile
 ```
-
-And then execute this code to generate the key:
+Next, configure the ETH Docker service. **Tip:** You can now call ethd from anywhere in your machine.
 
 ```
-touch secrets/encrypted_private_key.json
-docker compose run --rm ssv-node \
-/go/bin/ssvnode \
-generate-operator-keys \
--p password
+ethd config
 ```
+**Follow along the prompts in the terminal UI (TUI) to:**
+
+- Choose `Holešovice Testnet` >> `SSV node - consensus, execution and ssv-node`
+- Select `yes` for _Do you want to participate in DKG ceremonies as an operator_?
+- Once you see the screen below, select `<Cancel>` as we don't have our Operator ID yet.
 
-![image](https://hackmd.io/_uploads/HJNmMFVpA.png)
+![image](https://github.com/user-attachments/assets/b0dab641-8e46-4e04-af8d-585e50ba3fc8)
 
-To view the public key use this command:
+**Then copy your SSV node public key from your terminal output and save it on a text editor**
+
+![image](https://github.com/user-attachments/assets/b9dd7f94-e765-4696-b183-9c531cccd7a7)
+
+To re-print your SSV public key again, use this command:
 
 ```
+sudo apt install jq
 cat secrets/encrypted_private_key.json | jq .pubKey
 ```
 
 ![image](https://hackmd.io/_uploads/HJWsJoppA.png)
 
 **At this point, each operator must make a backup of the `encrypted_private_key.json` and the `password` files!**
 
+This is the most secure method but requires having SSH access to your server from your laptop's terminal (i.e., after manually adding SSH public keys to your server).
```

Comment:
I think this sentence can be clarified. Possibly add ssh access as a prerequisite at the top of the guide.